framework/helpers/BaseArrayHelper.php

```diff
@@ -625,6 +625,20 @@ public static function isIn($needle, $haystack, $strict = false)
     }
 
     /**
+     * Check whether a variable is an array or [[\Traversable]].
+     *
+     * This method does the same as the PHP function [is_array()](http://php.net/manual/en/function.is-array.php)
+     * but it does not only work for arrays but also objects that implement the [[\Traversable]] interface.
+     * @param mixed $var The variable being evaluated.
+     * @return boolean `true` if `$var` has an array-like value.
+     * @see http://php.net/manual/en/function.is_array.php
+     */
+    public static function isTraversable($var)
```

Comment:
I'd suggest to rename `$var` to `$value` because we try to avoid objective shortening